### PR TITLE
install with pip tools, add command

### PR DESCRIPTION
**Pull request by kisscelia**
Hi, @MihanEntalpo 

this pull request has two features:
1. can install, so we can use it as a  third package (now use python setup.py install)
2. add command: now name is allure2html, you can change it whatever you want.

Notice:
please change those arguments to others that you want, or told me here, then I can modify it.
PACKAGE_NAME, CMD, version, author, author_email, maintainer, maintainer_email, license... etc.
image

### CODE REVIEW

**Comment by kisscelia** [comment]
https://github.com/MihanEntalpo/allure-single-html-file/issues/10

**Comment by kisscelia** [comment]
Hi, @MihanEntalpo 
please review, thanks.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+def read(fname):
+    file_path = os.path.join(os.path.dirname(__file__), fname)
+    return codecs.open(file_path, encoding='utf-8').read()
+
+
+setup(
+    name=PACKAGE_NAME,
+    version='1.0.0',
+    author='Sine.wang',
+    author_email='sinecelia.wang@gmail.com',
```
Comment (MihanEntalpo) [inline setup.py:22]: If you don't mind, I'd ask you to add my username and email to the "author" and "author_email" fields space-separated with yours.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import codecs
+from setuptools import setup, find_packages
+
+
+PACKAGE_NAME = "allure2html"
+CMD = "allure2html"
```
Comment (MihanEntalpo) [inline setup.py:10]: I think that "allure2html" is a quite ambigous name for package (and, so, for command), I would recommend "allure-combine" or "allure2singlehtml"

**Review by MihanEntalpo — CHANGES_REQUESTED** [review]
Thanks for your contribution, that is a lot of work!
I have a pair of little changes

File: setup.py (hunk 1 above)
Comment (kisscelia) [reply setup.py:22]: Hi, @MihanEntalpo 
Yes, of course, I don't mind.  I will do that later.

File: setup.py (hunk 2 above)
Comment (kisscelia) [reply setup.py:10]: OK, sounds great.

File: README.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ python ./combine.py ./some/path/to/allure/generated/folder --remove-temp-files
 ## Import and use in python code
 
 ```python
-from combine import combine_allure
+from allure2html.combine import combine_allure
```
Comment (MihanEntalpo) [inline README.md:67]: By the way, how would the module be named in imports? We could use allure_combine or allurecombine (not allure-combine, unfortunatelly)

File: __init__.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from pkg_resources import get_distribution, DistributionNotFound
+
+
+try:
+    __version__ = get_distribution(__name__).version
+except DistributionNotFound:
+    # package is not installed
+    __version__ = "unknown"
+
+__pypi_url__ = "https://pypi.python.org/pypi/allure2html"
```
Comment (MihanEntalpo) [inline __init__.py:10]: Here is the old name of the module

**Review by MihanEntalpo — CHANGES_REQUESTED** [review]
Just a little naming issues

File: __init__.py (hunk 4 above)
Comment (kisscelia) [reply __init__.py:10]: OK, I'll change it.

File: README.md (hunk 3 above)
Comment (kisscelia) [reply README.md:67]: Yes, it will raise an error, when use allure-combine.

I will use allure_combine, is that OK?

Comment (kisscelia) [reply README.md:67]: > Yes, it will raise an error, when use allure-combine.
> 
> I will use allure_combine, is that OK?

or any other names?

Comment (MihanEntalpo) [reply README.md:67]: Ok

**Comment by kisscelia** [comment]
Hi, @MihanEntalpo 

please review, thanks.

**Comment by MihanEntalpo** [comment]
Ok, looking good. Some changes in readme.md could make it better, I'll make them by myself

**Comment by kisscelia** [comment]
> Ok, looking good. Some changes in readme.md could make it better, I'll make them by myself

OK

**Pull request merged**